secrethub.yml

```diff
@@ -0,0 +1,6 @@
+# You can replace these values with a path on SecretHub
```

Comment:
We can name this file `secrethub.env` later and perhaps use a variable prefix such as `$repo` in the paths to make this easy to reuse. Not for now though. The basic example works well 👍